spec/models/user_spec.rb

```diff
@@ -37,4 +37,11 @@
   it 'is not valid when naming 2 users with same username' do
     expect(user6.save).to be_falsey
   end
+
+  let(:user7) do
+    FactoryBot.build(:user)
+  end
+  it 'when creating a user it has the role of a user' do
+    expect(user7.role).to eq 'user'
+  end
```

Comment:
Let's add two separate describe blocks inside the Rspec block like:

describe 'validations' do
    it { is_expected.to validate_presence_of(:role) }
end

describe 'associations' do
  # add associations tests here
  it { is_expected.to have_many(:journals)
end

and complete the corresponding tests according to the model